src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -723,6 +724,15 @@ public AccessControlUser build(Object data) {
         accessControlUser.addKey(SafeEncoder.encode((byte[]) k));
       }
 
+      if (objectList.size() <= 9) {
+        // before redis 6.2, no channels info
+        return accessControlUser;
+      }
+      List<Object> channels = objectList.get(9);
+      for (Object channel : channels) {
+        accessControlUser.addChannel(SafeEncoder.encode((byte[]) channel));
+      }
```

Comment:
Use less return statements.
```suggestion
      // before redis 6.2, no channels info
      if (objectList.size() > 9) {
        List<Object> channels = objectList.get(9);
        for (Object channel : channels) {
          accessControlUser.addChannel(SafeEncoder.encode((byte[]) channel));
        }
      }
```